src/cuter_erlang.erl

```diff
@@ -220,6 +220,17 @@ safe_integer_to_list(I, Acc) ->
   D = int_to_char(I rem 10),
   safe_integer_to_list(I div 10, [D|Acc]).
 
+-spec list_to_integer([43 | 45 | 48..57, ...]) -> integer().
+list_to_integer([]) -> erlang:error(badarg);
+list_to_integer([$-|L]) -> -1 * list_to_integer_10(L, 0);
+list_to_integer([$+|L]) -> list_to_integer_10(L, 0);
+list_to_integer(L) -> list_to_integer_10(L, 0).
+
+list_to_integer_10([], Acc) -> Acc;
+list_to_integer_10([H|T], Acc) when H >= $0 andalso H =< $9 ->
```

Comment:
There is no reason to use `andalso` here.  This guard is simpler, and clearer I think, as `$0 =< H, H =< $9`.

But I will go ahead and merge this pull request nonetheless.